app/services/doc_auth/error_generator.rb

```diff
@@ -32,7 +32,6 @@ def initialize(config)
       'Control Number Crosscheck': { type: BACK, msg_key: Errors::CONTROL_NUMBER_CHECK },
       'Document Classification': { type: ID, msg_key: Errors::ID_NOT_RECOGNIZED },
       'Document Crosscheck Aggregation': { type: ID, msg_key: Errors::DOC_CROSSCHECK },
-      'Document Expired': { type: ID, msg_key: Errors::DOCUMENT_EXPIRED_CHECK },
```

Comment:
this should stay, this ticket is to remove the logic based on the error... the error will still be sent by the vendor and we still need to be able to handle it, translate messages, etc etc

Reply:
Updated.